src/wazuh_modules/task_manager/wm_task_manager.h

```diff
@@ -30,17 +30,6 @@ typedef enum _json_key {
     WM_TASK_ERROR_DATA
 } json_key;
```

Comment:
Couldn't be in a common place these keys? We could use them in the upgrade module to create the task messages.